scalableminds/webknossos
# In CI, disable escape characters in sbt logging

As described [in this SO thread](https://stackoverflow.com/questions/60236311/when-building-chisel-using-sbt-how-do-i-turn-off-the-progress-bars-etc-so-that), passing `-no-colors` to sbt will disable the escape characters in sbt logging that lead to the progress bar spam in the CI

### Steps to test:
- see sbt build steps in [CI run of this PR](https://app.circleci.com/pipelines/github/scalableminds/webknossos/3597/workflows/31c26cae-271d-41dc-9421-fb69a6d90e7e/jobs/14830)  should be readable, compare to [before](https://app.circleci.com/pipelines/github/scalableminds/webknossos/3596/workflows/f5a5252d-8775-4e0b-b2ff-f86374b0b3b7/jobs/14829)

### Issues:
- fixes #5535
